src/api/package/packages.ts

```diff
@@ -0,0 +1,43 @@
+export interface Packages {
+  packages: {
+    name: string;
+    version: string;
+    metadata: {
+      name: string;
+      description: string;
+      scope: string;
+      version: string;
+      keywords: string[];
+      date: string;
+      links: {
+        npm: string;
+        homepage: string;
+        repository: string;
+        bugs: string;
+      };
+      author: {
+        name: string;
+        url: string;
+      };
+      publisher: {
+        username: string;
+        email: string;
+      };
+      mantainers: {
+        username: string;
+        email: string;
+      }[];
```

Comment:
I believe this can actually just use the `Metadata` interface from  the adjacent `metadata.ts` file